usaspending_api/awards/models.py

```diff
@@ -84,6 +84,17 @@ class Meta:
         db_table = 'financial_accounts_by_awards_transaction_obligations'
 
 
+class AwardManager(models.Manager):
+    def get_queryset(self):
+        q_kwargs = {
+            "type": "U",
+            "total_obligation__isnull": True,
```

Comment:
A generated award will have these set to null, but will also receive no transactions. Thus, these will remain null. This finds those awards and throws them out. As soon as one of those awards gets a transaction (i.e. it is no longer empty), these will be updated via update_from_mod and the award will no longer match these criteria